wheatley/bot.py

```diff
@@ -69,6 +70,11 @@ def stage(self):
         return self._tower.number_of_bells
 
     # Callbacks
+    def _on_size_change(self):
+        if not self.row_generator.is_tower_size_valid(self._tower.number_of_bells):
+            print("WARNING: Row generation stage is NOT consistent with tower size! \
```

Comment:
Can this be a logger call? They get printed when when locally, but also the option to log to file when running on a server.

Can the message include the number of bells?
e.g. 
> Row generator stage requires {self.row_generator.number_of_bells} bells. Current tower size is {self._tower.number_of_bells}, Wheatley will crash when you go into changes unless something is done****

Reply:
Fixed!